src/s1reader/s1_reader.py

```diff
@@ -627,6 +640,96 @@ def get_ascending_node_time_orbit(orbit_state_vector_list: ET,
     return anx_time_orbit
 
 
+def get_osv_list_from_orbit(orbit_file: str | list,
```

Comment:
This function should just read in the orbits from the `orbit_file` or the list of orbit files

Reply:
Let's assume that if we get a list, it is sorted by start time